rgw/v2/tests/s3_swift/test_compression_encryption.py

```diff
@@ -0,0 +1,210 @@
+                            config.obj_size = size
+                            s3_object_name = utils.gen_s3_object_name(bucket.name, oc)
+                            log.info("s3 object name: %s" % s3_object_name)
+                            s3_object_path = os.path.join(
+                                TEST_DATA_PATH, s3_object_name
+                            )
+                            log.info("s3 object path: %s" % s3_object_path)
+                            log.info(f"print the {each_user}")
+                            if config.test_ops.get("upload_type") == "multipart":
+                                log.info("upload type: multipart")
+                                reusable.upload_mutipart_object(
+                                    s3_object_name,
+                                    bucket,
+                                    TEST_DATA_PATH,
+                                    config,
+                                    each_user,
+                                )
+                            else:
+                                log.info("upload type: normal")
+                                reusable.upload_object(
+                                    s3_object_name,
+                                    bucket,
+                                    TEST_DATA_PATH,
+                                    config,
+                                    each_user,
+                                )
+
+    if config.test_ops["server_side_copy"] is True:
+        bucket1, bucket2 = buckets_created
+
+        # copy object1 from bucket1 to bucket2 with the same name as in bucket1
+        log.info("copying first object from bucket1 to bucket2")
+        all_keys_in_buck1 = []
+        for obj in bucket1.objects.all():
+            all_keys_in_buck1.append(obj.key)
+        copy_objects = []
+        for object_name in all_keys_in_buck1:
+            copy_object_name = object_name + "_copied_obj"
+            copy_objects.append(copy_object_name)
+            log.info(f"copy object name: {copy_object_name}")
+            s3_client.copy_object(
+                Bucket=bucket2.name,
+                Key=copy_object_name,
+                CopySource={
+                    "Bucket": bucket1.name,
+                    "Key": object_name,
+                },
+            )
+
+        # list the objects in bucket2
+        log.info("listing all objects im bucket2 after copy")
+        all_bucket2_objs = []
+        for obj in bucket2.objects.all():
+            log.info(obj.key)
+            all_bucket2_objs.append(obj.key)
+
+        # check for object existence in bucket2
+        for copy_object_name in copy_objects:
+            if copy_object_name in all_bucket2_objs:
+                log.info("server side copy successful")
+            else:
+                raise TestExecError("server side copy operation was not successful")
+
+    if config.test_ops["remote_object_download"] is True:
+        each_user = user1
```

Comment:
@viduship , I think user1 in this line is not defined, could you please check once

Reply:
hi @hmaheswa , I have removed the remote download case as of now and only kept the s3 copy one.

Please review